bot.go

```diff
@@ -52,6 +55,28 @@ func (bot *Bot) AddCron(c crons.Cron) {
 	})
 }
 
+func (bot *Bot) CheckVersion(sigChan chan os.Signal) {
```

Comment:
```suggestion
func (bot *Bot) SetCheckVersionCron(sigChan chan os.Signal) {
```
함수 이름 좀 더 명시적으로 해줘도 될듯 합니다? 아니면 어떤함수인지 주석으로 적어줘도 좋을것 같고요